src/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -22,11 +23,11 @@ public sealed class LicenseManager
     {
         private static readonly object s_selfLock = new object();
 
-        private static volatile LicenseContext s_context;
+        private static LicenseContext s_context;
         private static object s_contextLockHolder;
         private static volatile Hashtable s_providers;
         private static volatile Hashtable s_providerInstances;
-        private static readonly object s_internalSyncObject = new object();
+        private static object s_internalSyncObject = new object();
```

Comment:
Sorry, when I wrote "let's just revert the changes here", I meant in the whole file.  There are many other uses of s_internalSyncObject throughout the file, and I would rather we have the safety of that being readonly that save a few lines of code by using EnsureInitialized.